int8 motor_id
bool enabled
float32 velocity
